example/site.clj

```diff
@@ -13,7 +33,15 @@
     (stylesheet global-style)]
    [:body content]])
 
+(def-asset about-page
+  {:path "/about.html"
+   :type :html
+   :data (page "Sur Moi" (-> about-page-org-string
+                             ;; TODO Can't slurp a file with sci, what do?
+                             (clj-org.org/parse-org)
+                             (:content)))})
+
 (def-asset home-page
   {:path "/index.html"
    :type :html
-   :data (page "Hello world" [:p "This is magic"])})
+   :data (page "Hello world" [:a {:href "/about.html"} "This is a blog link."])})
```

Comment:
Prefer using the built in `asset-path` function instead of manually writing the path. Eg:

```clj
(page "Hello world" [:a {:href (asset-path about-page)} "This is a blog link"])
```